src/hyp3_isce2/etc/download_example.py

```diff
@@ -51,5 +52,8 @@
     download_metadata(session, sec_desc, 'secondary_descending.xml')
 
 # Example 2
-# Download with SAFE spoofing
-download_bursts([ref_desc, sec_desc])
+# Download with the burst2safe package
+ref = 'S1_136231_IW2_20200604T022312_VV_7C85-BURST'
+sec = 'S1_136231_IW2_20200616T022313_VV_5D11-BURST'
+
+burst2safe.burst2safe([ref, sec], polarizations=['VV'], all_anns=True)
```

Comment:
```suggestion
burst2safe.burst2safe([ref])
burst2safe.burst2safe([sec])
```
Interface is a little different for `burst2safe`